package.json

```diff
@@ -12,11 +12,14 @@
     "debug-unit": "DEBUG_TEST=true node --inspect-brk ./node_modules/.bin/jasmine test/test.js",
     "test-phantom": "python third_party/phantomjs/test/run-tests.py",
     "test-doclint": "jasmine utils/doclint/check_public_api/test/test.js && jasmine utils/doclint/preprocessor/test.js",
-    "test": "npm run lint --silent && npm run coverage && npm run test-phantom && npm run test-doclint",
+    "test": "npm run lint --silent && npm run coverage && npm run test-phantom && npm run test-doclint && npm run node6 &&  npm run test-node6",
```

Comment:
not a big deal, but I noticed that there's about a half second overhead for executing an npm script, so when you chain 5 npm script commands it's like 0.5s * 5 = 2.5sec overhead.